server/v2/store/server.go

```diff
@@ -0,0 +1,66 @@
+package store
```

Comment:
if we were following the pattern of other server components defined under `server/v2` this would be its own go.mod, but it's a part of `cosmossdk.io/server/v2` which is inconsistent and a bit confusing.  would it make sense to define the server component in `store/v2` ?